site/content/blog/get-started-with-opentitan.md

```diff
@@ -0,0 +1,12 @@
++++
+date = "2019-11-26T9:00:00+00:00"
+title = "Get started with OpenTitan"
+author = "Greg Chadwick"
++++
+
+
+Interested in trying out the recently announced OpenTitan? We've put together a video that goes through an overview of how the OpenTitan system (named Earl Grey) is put together and how to get up and running with our pre-built release (providing simulator binaries and pre-built FPGA images for the Nexys Video Artix-7 board). It follows the steps from the [OpenTitan Quickstart Guide](https://docs.opentitan.org/doc/ug/quickstart/).
```

Comment:
Is it worth mentioning the "Earl Grey" name in this post? Even if so, maybe it's worth saying something like "The current OpenTitan prototype system" or similar?